README.md

```diff
@@ -174,7 +174,7 @@ author:
 
 All those questions can lead to bugs if they're not properly answered, and
 [documentation never answers them
-reliably](#112-vague-and-bloated-documentation))
+reliably](#112-vague-and-bloated-documentation)
```

Comment:
Please add a period to the end of this sentence (ideally outside the link) to replace the inappropriate closing parenthesis.

Reply:
Done